### PR TITLE
Update organ_types.yaml

### PR DESCRIPTION
**Pull request by icaoberg**
Added UBERON terms for most of the missing organs. Needed for CFDE submission.

### CODE REVIEW

**Comment by yuanzhou** [comment]
Are you going to test your changes on DEV first? We are in the process of making github branch changes. You may want to change the base branch to `dev-integrate` if meant to be tested on DEV first. Then another PR from your original branch into `main` for TEST infrastructure and release purposes.

File: src/search-schema/data/definitions/enums/organ_types.yaml [hunk 1]
```diff
@@ -10,12 +10,16 @@
 
 AO:
   description: Aorta
+  iri: http://purl.obolibrary.org/obo/UBERON_0000947
 BL:
   description: Bladder
+  iri: http://purl.obolibrary.org/obo/UBERON_0018707
```
Comment (bhonick) [inline src/search-schema/data/definitions/enums/organ_types.yaml:16]: Would the iri for ["Urinary Bladder"(https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0001255) work better here? It is more specific than "Bladder organ"

File: src/search-schema/data/definitions/enums/organ_types.yaml [hunk 2]
```diff
@@ -28,8 +32,10 @@ HT:
   iri: http://purl.obolibrary.org/obo/UBERON_0000948
```
Comment (bhonick) [inline src/search-schema/data/definitions/enums/organ_types.yaml:32]: Per Mariah's comment in the Slack thread, I suggest using the UBERON entries for [right uterine tube](http://purl.obolibrary.org/obo/UBERON_0001302) and [left uterine tube](http://purl.obolibrary.org/obo/UBERON_0001303) for the left and right fallopian tubes.

File: src/search-schema/data/definitions/enums/organ_types.yaml [hunk 3]
```diff
@@ -43,24 +49,30 @@ LN:
   description: Knee (Left)
```
Comment (bhonick) [inline src/search-schema/data/definitions/enums/organ_types.yaml:49]: http://purl.obolibrary.org/obo/UBERON_0001465

File: src/search-schema/data/definitions/enums/organ_types.yaml [hunk 4]
```diff
@@ -71,6 +83,7 @@ RN:
   description: Knee (Right)
```
Comment (bhonick) [inline src/search-schema/data/definitions/enums/organ_types.yaml:83]: http://purl.obolibrary.org/obo/UBERON_0001465

**Review by bhonick — COMMENTED** [review]
See highlighted comments

File: src/search-schema/data/definitions/enums/organ_types.yaml (hunk 1 above)
Comment (icaoberg) [reply src/search-schema/data/definitions/enums/organ_types.yaml:16]: @pdblood @computationdoc I used the term for `Bladder` but @bhonick wants to know if `Urinary Bladder` is more acceptable/proper/correct. What do you think?

File: src/search-schema/data/definitions/enums/organ_types.yaml (hunk 2 above)
Comment (icaoberg) [reply src/search-schema/data/definitions/enums/organ_types.yaml:32]: @pdblood @computationdoc similar inquiry as before. What is the proper/correct entry to be used?

File: src/search-schema/data/definitions/enums/organ_types.yaml (hunk 1 above)
Comment (bherr2) [reply src/search-schema/data/definitions/enums/organ_types.yaml:16]: And as an FYI, here are all the mappings we have from CCF-HRA Reference Organ to uberon term: <http://grlc.io/api-git/hubmapconsortium/ccf-grlc/subdir/ccf/ref-organ-terms>

**Review by bhonick — APPROVED** [review]

File: src/search-schema/data/definitions/enums/organ_types.yaml (hunk 1 above)
Comment (icaoberg) [reply src/search-schema/data/definitions/enums/organ_types.yaml:16]: @bherr2 @bhonick I understand your suggestion but I need to know what is the action item. Should I keep `Bladder` or use `Urinary Bladder`? Whatever either/both of you think is best cc @pdblood

Comment (bhonick) [reply src/search-schema/data/definitions/enums/organ_types.yaml:16]: @icaoberg I would recommend using the most specific descriptor for human anatomy. Therefore, `Urinary Bladder` would be the way to go.

**Comment by yuanzhou** [comment]
@icaoberg since this PR has been there for a while, can you sync your branch with the latest `main` branch just so you have the latest code?

File: src/search-schema/data/definitions/enums/organ_types.yaml (hunk 1 above)
Comment (icaoberg) [reply src/search-schema/data/definitions/enums/organ_types.yaml]: Resolved as per @bhonick suggestion.

File: src/search-schema/data/definitions/enums/organ_types.yaml (hunk 2 above)
Comment (icaoberg) [reply src/search-schema/data/definitions/enums/organ_types.yaml:34]: Updated as per @bhonick and @MariahKenney suggestion.

**Comment by icaoberg** [comment]
@bhonick @bherr2 made the suggested changes.

**Comment by yuanzhou** [comment]
Closing this PR since all the assay types updates should go to UBKG. FYI @icaoberg and @AlanSimmons .

**Pull request closed without merging**